src/lib/Scenes/Home/Components/ForYou/Components/FairsRail.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from "react"
```

Comment:
we should use the following to import react from now on.
`import React from "react"`

I explain the reasoning in this PR (https://github.com/artsy/emission/pull/820)